conf/app.routes

```diff
@@ -1,17 +1,18 @@
 # microservice specific routes
 
-
 # subscription field definitions endpoints routes
+PUT           /definition/context/:apiContext/version/:apiVersion                      @uk.gov.hmrc.apisubscriptionfields.controller.FieldsDefinitionController.upsertFieldsDefinition(apiContext: String, apiVersion: String)
 
-# TODO
+GET           /definition/context/:apiContext/version/:apiVersion                      @uk.gov.hmrc.apisubscriptionfields.controller.FieldsDefinitionController.getFieldsDefinition(apiContext: String, apiVersion: String)
 
```

Comment:
Could you please add a new route?
```
GET     /definition/
```

This would need to return a JSON with all field definitions